drv/vsc85xx/src/atom.rs

```diff
@@ -8,9 +8,7 @@ use vsc7448_pac::phy;
 use vsc_err::VscError;
 
 /// Based on `vtss_atom_patch_suspend` in the SDK
-pub fn atom_patch_suspend<'a, 'b, P: PhyRw>(
-    phy: &'b mut Phy<'a, P>,
-) -> Result<(), VscError> {
+pub fn atom_patch_suspend<P: PhyRw>(phy: &mut Phy<P>) -> Result<(), VscError> {
```

Comment:
I think you want `Phy<'_, P>` here, and probably also to set `deny(elided_lifetimes_in_paths)` at the crate root. I wish there was a way we could set this at the workspace level, but, there does not appear to be.

Reply:
I ended up turning it on as a Clippy lint (in `cargo xtask clippy`), then fixed it _everywhere_!